README.md

```diff
@@ -208,6 +208,7 @@ Shaka Player supports:
     - With help from [mux.js][] v5.6.3+, supported embedded in TS
   - CEA-708
     - Supported embedded in MP4
+  - SubRip (SRT)
```

Comment:
Since we use `fromUTF8` in the SRT parser, we are inherently assuming that the input is UTF8-encoded.  So I think it would be best to specify here that we accept only UTF8 SRTs as input.

I take your point about TextDecoder being on the table now, but that is not going to affect our parser unless the user specifies an encoding somehow and we then use that information to call into TextDecoder directly.  An app could possibly use TextDecoder and TextEncoder to convert the input, though, and pass the resulting SRT file as a data URI.

Furthermore, polyfills for TextDecoder will likely only support UTF8 input.  So even if we did use TextDecoder and a user-supplied encoding parameter directly in the SRT parser, we would still have to handle failures on any encoding other than UTF8.

So probably best for now just to specify that we only take UTF8.

Reply:
I added it to the README